homework/list.js

```diff
@@ -1,44 +1,36 @@
 'use strict';
 
-(function() {
+(function(exports) {
 
+var ListManager = function() {
   var _listNoteContent = [];
   var _wrapper = document.querySelector('#note-list-wrapper');
+};
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+ListManager.prototype = {  
+  onNoteOpen(event) {
```

Comment:
It's a convention to prefix method with `on` and name it as the corresponding event. Therefore, this handler should be renamed as `onClick`.